docs/textile-js-sdk/0.1.0/Connection.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
```

Comment:
Should probs change the path names here to reflect the 'new' name: `docs/js-textile-http-client/0.1.0/...`?

Reply:
I'm assuming these are auto-generated, but couldn't find a better place to make that comment 🤓